lib/SIL/Utils/OwnershipUtils.cpp

```diff
@@ -56,6 +56,13 @@ bool swift::findPointerEscape(SILValue original) {
         worklist.pushIfNotVisited(phi);
         break;
       }
+      case OperandOwnership::Borrow: {
+        auto borrowOp = BorrowingOperand(use);
+        if (auto borrowValue = borrowOp.getBorrowIntroducingUserResult()) {
```

Comment:
Can a yielded or captured value escape? I think we talked about this being illegal because it crosses a function boundary. But the implementation here doesn't explain why we're ignoring these cases. We should at least explain that it was intentional.